packages/document-drive/package.json

```diff
@@ -49,7 +49,8 @@
     "bench": "vitest bench",
     "migrate": "prisma generate && prisma db push",
     "copy-prisma": "copyfiles prisma/schema.prisma dist/ && copyfiles 'src/storage/prisma/client/**/*' dist/",
-    "prisma-generate": "prisma generate --no-hints && npm run copy-prisma"
+    "prisma-generate": "prisma generate --no-hints && npm run copy-prisma",
+    "build": "npm run copy-prisma"
```

Comment:
@ryanwolhuter This should be `build:misc` right? But it might need to run before `tsc` in this case.

Should all packages have a `build` script similar to the `build:all` on the root package.json?

Reply:
@acaldas this should actually have been removed. we decided to stop re-generating the prisma stuff the whole time a while ago, so there should be no need to run this command at all